### PR TITLE
fix: UI issues in dart mode.

### PR DESCRIPTION
**Pull request by akarsh-jain-790**
Fixes: #110

Screenshot 2023-05-15 at 4 26 29 PM

Screenshot 2023-05-15 at 4 26 15 PM

Screenshot 2023-05-15 at 2 53 13 PM

Screenshot 2023-05-15 at 2 53 25 PM

### CODE REVIEW

**Comment by akarsh-jain-790** [comment]
It has been a while, so just bumping this up for quick feedback from @annieappflowy @hyj1204

File: lib/src/render/action_menu/action_menu_item.dart [hunk 1]
```diff
@@ -84,8 +84,8 @@ class ActionMenuItemWidget extends StatelessWidget {
     final editorStyle = EditorStyle.of(context);
```
[the remark this answers is not in the document]
Comment (akarsh-jain-790) [reply lib/src/render/action_menu/action_menu_item.dart]: @hyj1204 I have implemented your suggested changes, you can re-review now.

File: lib/src/render/action_menu/action_menu_item.dart [hunk 2]
```diff
@@ -81,11 +81,10 @@ class ActionMenuItemWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final editorStyle = EditorStyle.of(context);
     final isSelected = item.selected?.call() ?? false;
     final color = isSelected
-        ? editorStyle?.selectionMenuItemSelectedIconColor
-        : editorStyle?.selectionMenuItemIconColor;
+        ? Theme.of(context).colorScheme.primary
+        : Theme.of(context).colorScheme.tertiary;
```
[the remark this answers is not in the document]
Comment (akarsh-jain-790) [reply lib/src/render/action_menu/action_menu_item.dart:87]: @hyj1204  specified in the issue that we should use all the colors from the current ThemeData (Theme.of(context).***) and not from the EditorStyle, as it is currently being refactored.

**Review by hyj1204 — APPROVED** [review]
LGTM

File: lib/src/render/action_menu/action_menu_item.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (Xazin) [reply lib/src/render/action_menu/action_menu_item.dart:87]: That makes sense.

**Pull request merged**